internal/templates/funcmap.go

```diff
@@ -10,12 +11,33 @@ import (
 )
 
 // GetFuncMap returns the list of functions provided by sprig. It adds the
-// function "toTime" and changes the function "fail".
+// functions "formatTime", "toTime", "mustToTime", and changes the function
+// "fail".
```

Comment:
Might want to add `toLayout` (or its updated name(s)) here

Reply:
Fixed with a23cb16